app/views/licences/layout.njk

```diff
@@ -18,62 +18,66 @@
-          Summary
-        </a>
-      </li>
-      <li
-        class="govuk-tabs__list-item {% if activeTab === 'contact-details' %} govuk-tabs__list-item--selected {% endif %}"
-        data-test="#tab_contact">
-        <a class="govuk-tabs__tab" href="contact-details">
-          Contact details
-        </a>
-      </li>
-      <li class="govuk-tabs__list-item {% if activeTab === 'returns' %} govuk-tabs__list-item--selected {% endif %}"
-          data-test="#tab_returns">
-        <a class="govuk-tabs__tab" href="returns">
-          Returns
-        </a>
-      </li>
-      <li
-        class="govuk-tabs__list-item {% if activeTab === 'communications' %} govuk-tabs__list-item--selected {% endif %}"
-        data-test="#tab_communications">
-        <a class="govuk-tabs__tab" href="communications">
-          Communications
-        </a>
-      </li>
-      {% if 'billing' in roles %}
-        <li class="govuk-tabs__list-item {% if activeTab === 'bills' %} govuk-tabs__list-item--selected {% endif %}"
-            data-test="#tab_bills">
-          <a class="govuk-tabs__tab" href="bills">
-            Bills
-          </a>
-        </li>
-      {% endif %}
-      {% if 'view_charge_versions' in roles %}
-        <li class="govuk-tabs__list-item {% if activeTab === 'set-up' %} govuk-tabs__list-item--selected {% endif %}"
-            data-test="#tab_licence_set_up">
-          <a class="govuk-tabs__tab" href="set-up">
-            Licence set up
-          </a>
-        </li>
-      {% endif %}
-    </ul>
+          <li
+            class="x-govuk-sub-navigation__section-item {% if activeTab === 'communications' %}x-govuk-sub-navigation__section-item--current{% endif %}">
+            <a class="x-govuk-sub-navigation__link" href="communications"
+              aria-current="{{ 'page' if activeTab === 'communications' }}">
+              Communications
+            </a>
+          </li>
 
-    <div class="govuk-tabs__panel" id="{{ activeTab }}">
+          {% if 'billing' in roles %}
+            <li
+              class="x-govuk-sub-navigation__section-item {% if activeTab === 'bills' %}x-govuk-sub-navigation__section-item--current{% endif %}">
+              <a class="x-govuk-sub-navigation__link" href="bills" aria-current="{{ 'page' if activeTab === 'bills' }}">
+                Bills
+              </a>
+            </li>
+          {% endif %}
+
+          {% if 'view_charge_versions' in roles %}
+            <li
+              class="x-govuk-sub-navigation__section-item {% if activeTab === 'set-up' %}x-govuk-sub-navigation__section-item--current{% endif %}">
+              <a class="x-govuk-sub-navigation__link" href="set-up"
+                aria-current="{{ 'page' if activeTab === 'set-up' }}">
+                Licence set up
+              </a>
+            </li>
+          {% endif %}
+        </ul>
+      </nav>
+    </div>
+    <div class="govuk-grid-column-three-quarters">
       {% block subPageContent %}
```

Comment:
All sub page content needs a new heading - we can use the page title here - they needs to now be H1

Reply:
We could overide the base page headting html by setting it to empty ?  then it would not show ?